config/config.go

```diff
@@ -141,18 +141,12 @@ func NewDefaultConfig() *AgentConfig {
 			},
 		},
 		CMAB: CMABConfig{
-			Enabled:            false,
-			PredictionEndpoint: "https://prediction.cmab.optimizely.com",
+			PredictionEndpoint: "https://prediction.cmab.optimizely.com/predict/%s",
```

Comment:
Wondering how we use these cmab configs with go-sdk cores. Planned in a separate PR?